README.md

```diff
@@ -28,65 +28,34 @@ USAGE
 <!-- usagestop -->
 # Commands
 <!-- commands -->
-* [`lexcmdcli list`](#lexcmdcli-list)
-* [`lexcmdcli build`](#lexcmdcli-build)
+* [`lexcmdcli lexcmdcli build -b <BOT_ID> -a <ALIAS_NAME>`](#lexcmdcli-lexcmdcli-build--b-bot_id--a-alias_name)
```

Comment:
It looks like there is double `lexcmdcli` here....

Reply:
Great find, thanks!